concise-mid-tag.cddl

```diff
@@ -136,8 +136,14 @@ tagged-ueid-type = #6.48000(ueid-type)
 impl-id-type = bytes .size 32
 tagged-impl-id-type = #6.47115(impl-id-type)
 
+$measured-element-type-choice /= tstr
```

Comment:
Can we add a type choice for tagged-oid and tagged-uuid? That would allow matching the 'type' value for DiceTcbInfos when there are a sequence of tcbinfo in the same certificate. 

Additionally, do we anticipate, that if the 'environment-map' / 'class-id' layer and index are specified that these would apply to all measurement-elements? That seems to align with the usage in tcbinfo where a sequence of tcbinfos repeats the vendor, layer, index for all tcbinfos in the same dice cert.

Reply:
> Can we add a type choice for tagged-oid and tagged-uuid? That would allow matching the 'type' value for DiceTcbInfos when there are a sequence of tcbinfo in the same certificate.

sure!

> Additionally, do we anticipate, that if the 'environment-map' / 'class-id' layer and index are specified that these would apply to all measurement-elements? That seems to align with the usage in tcbinfo where a sequence of tcbinfos repeats the vendor, layer, index for all tcbinfos in the same dice cert.

What do you mean by `measurement-elements` -- looks as a funny hybrid between `measured-elements` and `measurement-values-map` :-)

Reply:
For the sake of terminology 'environment' refers to something identified by environment-map. 'component' refers to something that has a measurement value that is different from another component within the same environment. 

Do we agree that the 'layer' and 'index' values are attributes of the environment and not attributes of a component?

Reply:
> Do we agree that the 'layer' and 'index' values are attributes of the environment and not attributes of a component?

100%